### PR TITLE
[v10] bump go mod go1.18

### PR DESCRIPTION
**Pull request by lxea**
Closes #13958

### CODE REVIEW

**Review by reedloden — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -54,9 +54,7 @@ require (
 	github.com/gravitational/configure v0.0.0-20180808141939-c3428bd84c23
 	github.com/gravitational/form v0.0.0-20151109031454-c4048f792f70
 	github.com/gravitational/kingpin v2.1.11-0.20220708173555-cb79b87d008b+incompatible
-	github.com/gravitational/license v0.0.0-20210218173955-6d8fb49b117a
```
Comment (marcoandredinis) [inline go.mod:57]: Those should not be removed
It usually happens when you don't have the `e` submodule

File: go.mod [hunk 2]
```diff
@@ -54,9 +54,7 @@ require (
...
 	github.com/gravitational/oxy v0.0.0-20211213172937-a1ba0900a4c9
-	github.com/gravitational/reporting v0.0.0-20210923183620-237377721140
```
Comment (reedloden) [inline go.mod:59]: Same here as above

File: go.sum [hunk 3]
```diff
@@ -609,8 +609,6 @@ github.com/gravitational/httprouter v1.3.1-0.20220408074523-c876c5e705a5 h1:qg8F
 github.com/gravitational/httprouter v1.3.1-0.20220408074523-c876c5e705a5/go.mod h1:JR6WtHb+2LUe8TCKY3cZOxFyyO8IZAc4RVcycCCAKdM=
 github.com/gravitational/kingpin v2.1.11-0.20220708173555-cb79b87d008b+incompatible h1:c5+i6ThhWLYKtipKjQ+UP/816/GCmo8Zx8qMXp+hf+A=
 github.com/gravitational/kingpin v2.1.11-0.20220708173555-cb79b87d008b+incompatible/go.mod h1:LWxG30M3FcrjhOn3T4zz7JmBoQJ45MWZmOXgy9Ganoc=
-github.com/gravitational/license v0.0.0-20210218173955-6d8fb49b117a h1:PN5vAN1ZA0zqdpM6wNdx6+bkdlQ5fImd75oaIHSbOhY=
```
Comment (reedloden) [inline go.sum:612]: Same as above

File: go.sum [hunk 4]
```diff
@@ -621,14 +619,11 @@ github.com/gravitational/protobuf v1.3.2-0.20201123192827-2b9fcfaffcbf h1:MQ4e8X
 github.com/gravitational/protobuf v1.3.2-0.20201123192827-2b9fcfaffcbf/go.mod h1:SlYgWuQ5SjCEi6WLHjHCa1yvBfUnHcTbrrZtXPKa29o=
 github.com/gravitational/redis/v8 v8.11.5-0.20220211010318-7af711b76a91 h1:LWtt0fv7KDZt6ykJGBXd/04sONaHIMXFzHvmUAqhG8c=
 github.com/gravitational/redis/v8 v8.11.5-0.20220211010318-7af711b76a91/go.mod h1:oTV66KM4DNr2OWoOP4HxexJUPZxPkXHpi6DeLSDsCnA=
-github.com/gravitational/reporting v0.0.0-20210923183620-237377721140 h1:UACNXsuOXxe2dpjn3SA2g4YUgOoZTxQ6yf+QYAYcQMc=
```
Comment (reedloden) [inline go.sum:624]: Same as above

**Review by reedloden — CHANGES_REQUESTED** [review]

**Review by reedloden — APPROVED** [review]

**Review by marcoandredinis — APPROVED** [review]

File: go.mod [hunk 5]
```diff
@@ -289,7 +292,7 @@ require (
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
 	go.uber.org/multierr v1.6.0 // indirect
 	go.uber.org/zap v1.19.1 // indirect
-	golang.org/x/exp v0.0.0-20220613132600-b0d781184e0d // indirect
+	golang.org/x/exp v0.0.0-20220613132600-b0d781184e0d
```
Comment (espadolini) [inline go.mod:295]: Missing a `go mod tidy` here; if it doesn't change anything, put all the required imports in the same group and run it again.

**Review by espadolini — APPROVED** [review]

**Review by tcsc — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

**Pull request merged**